vpt/processors/gaze_detector/__init__.py

```diff
@@ -92,7 +92,7 @@ def get_square_box(box):
 
         if diff == 0:  # Already a square.
             return box
-        elif diff > 0:  # Height > width, a slim box.
+        if diff > 0:   # Height > width, a slim box.
```

Comment:
Why this?

Reply:
Just like with the `break`, there is a `return` in the `if` branch here, meaning the `else` is useless

Reply:
Got it